src/Settings.hpp

```diff
@@ -22,6 +22,9 @@ class Settings
         bool   getHeaderBar() const;
         void   setStartInTray(bool enable);
         bool   getStartInTray() const;
+        void   setAutostart(bool autostart);
+        bool   getAutostart();
+        void   autostart();
```

Comment:
Move this logic into `setAutoStart` so that the file is copied/removed everytime `setAutoStart` is called.

Reply:
This operation could be performed only once while program is closing. That is why we call this function at `MainWindow` dtor. Executing each time when user toggle the button is unnecessary. But to keep the structure of Setting class i can change it. What do you think about it?

Reply:
Well, setAutoStart function itself should do what its name refers, it should check if the file exists or not and make the appropriate copy/remove. If you think it will be called too many times when the switch is toggled which is possible, we can only call this function in the dtor of MainWindow. Otherwise, I don't think that will be a big problem since it's only one file and it's under user's responsibility to not abuse it.